controller/src/main/java/io/pravega/controller/store/stream/tables/HistoryIndexRecord.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.controller.store.stream.tables;
+
+import io.pravega.common.util.BitConverter;
+import lombok.Data;
+
+import java.util.Optional;
+
+@Data
+/**
+ * Class corresponding to a record/row in Index table.
+ * Each row is fixed size
+ * Row: [eventTime, pointer-into-history-table]
+ */
+public class HistoryIndexRecord {
```

Comment:
Are we supposed to be getting rid of `IndexRecord` in this PR? I don't see it being removed.

Reply:
Index Record is replaced with HistoryIndexRecord